apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -924,6 +924,7 @@ export default Vue.extend({
       this.rawTableKeys = await this.connection.getTableKeys(this.table.name, this.table.schema)
       const rawPrimaryKeys = await this.connection.getPrimaryKeys(this.table.name, this.table.schema);
       this.primaryKeys = rawPrimaryKeys.map((key) => key.columnName);
+      this.filters = normalizeFilters(this.initialFilters || [])
```

Comment:
Did we need to do this previously? I don't think we did, shouldn't `initialFilter` in the tabulator constructor do this?

Reply:
Actually, we [had](https://github.com/beekeeper-studio/beekeeper-studio/blob/e4c8e713296b91045b361da076e4426a294932a9/apps/studio/src/components/tableview/TableTable.vue#L1124-L1126) this before but in a little different way. `this.filter` was used in [filterForTabulator()](https://github.com/beekeeper-studio/beekeeper-studio/blob/e4c8e713296b91045b361da076e4426a294932a9/apps/studio/src/components/tableview/TableTable.vue#L858) which then used in many different parts including the tabulator [dataFetch](https://github.com/beekeeper-studio/beekeeper-studio/blob/e4c8e713296b91045b361da076e4426a294932a9/apps/studio/src/components/tableview/TableTable.vue#L1656).

In this case, `this.filters` is empty and was not attached to the `initialFilters` initially, which is why when we initialize the tabulator, the data is not filtered at all.

Reply:
I accidentally removed that part and didn't know it's actually gonna affect the `initialFilter`.